### ISSUE TITLE
Found binding \xc behaves incorrectly with custom prompt, identical custom binding behaves differently

### ISSUE THREAD

**Issue by langenhagen** [fish-shell/fish-shell#7101]
fish version 3.1.0
OS: Ubuntu 20.04
`uname -a`: Linux 5.4.0-33-generic #37-Ubuntu SMP Thu May 21 12:53:59 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
$TERM=xterm-256color
- I have `oh-my-fish` installed

The issue does not happen in a clean fish, i.e. it does not occur when running `sh -c 'env HOME=$(mktemp -d) fish'`

1. define a different prompt that omits the username and the host name
2. check the binding for ctrl + x `bind \cx` for copying the command line
3. create an identical user binding on another key combo. I used `\ex`: `bind \ex fish_clipboard_copy`
4. test the bindings and compare the clipboards, also in different folders, e.g. within a git repo and outside a git repo.
For me, the results differ, although I believe they should be identical.

I made a asciinema video to showcase the issue: https://asciinema.org/a/OCFZOxGDhfaS2v3poZ5KbLPMI

**Comment by faho** [comment]
Okay, some questions:

- What does pressing ctrl-x send in your terminal? Start `fish_key_reader` and press ctrl-x

- Does this also happen without oh-my-fish installed? That does a *whole load of stuff* that might be interfering here.

- How do you *paste* the things? Can you confirm it really is the copying that's at fault here, not that you're e.g. pasting with ctrl-shift-v with one binding and with ctrl-v with the other? If you paste into a different application, does that show the fault?

- Do you use vi-mode? If yes, in which mode do you copy?

**Comment by langenhagen** [comment]
Thx for the response, @faho . I was able to resolve the issue with your input. Indeed, fish shell works nicely. `tmux` shadowed ctrl+x and was performing a clipboard action instead of fish. I called `fish_key_reader` which ignored ctrl + x as input and kept waiting for input. I then realized that `tmux` might have blocked the input. I removed the according setting from my `.tmux.conf` and enjoy now fish's default `ctrl + x` binding.
